# Acelaração gravitacional

## Conteúdo

### Física da Terra

#### Resumo teórico de Gravitação
Uso da Lei da Gravitação



### 1. A Lei Universal da Gravitação

Um corpo de massa $m$ em movimento possui um momento de inércia $(m\mathbf{v})$. Para mudarmos este movimento é necessário aplicarmos uma força 
$\mathbf{F}$ a este corpo. A segunda lei de movimento de Newton estabelece que a taxa de variação do momento de uma massa é proporcional a força que atua sobre ela, e acontece na direção da força. Para o caso de massas constantes esta lei serve como definição de força $\mathbf{F}$ em termos da aceleração 
$(\mathbf{a})$ dada e pela massa $(m)$:

\begin{equation*}
    \mathbf{F} = \frac{d}{dt} (m\mathbf{v}) = m \frac{d\mathbf{v}}{dt}= m \mathbf{a}
    \label{newton_lei}\tag{1}
\end{equation*}

A unidade de força no sistema SI é o Newton (N). Ela é definida como sendo a força que dá a uma massa de um quilograma, uma aceleração de $1\, \mathrm{m}/\mathrm{s}^2$.

Nós conhecemos a célebre observação de Newton da maçã em queda, a qual ele relacionou com a atração gravitacional que a Terra exercia sobre a maçã. Entretanto, a genialidade de Newton foi reconhecer que o campo gravitacional que faz com que a maçã caia é a mesma que mantém a Lua em órbita em torno da Terra e que mantém os planetas girando ao redor do Sol. Newton deduziu que a atração gravitacional $\mathbf{F}$ entre duas partículas de massas $m$ e $M$, separadas pela distância $r$, é proporcional ao produto destas massas e inversamente proporcional ao quadrado da distância entre elas:

\begin{equation*}
    \mathbf{F} = -G\frac{m M}{r^2} \hat{\mathbf{r}}
    \label{grav_lei}\tag{2}
\end{equation*}

Onde $\hat{\mathbf{r}}$ é o vetor unitário na direção da coordenada $r$, direcionada para fora do centro de referência da massa $M$. O sinal negativo indica que a força $\mathbf{F}$ age na direção oposta, em direção da massa $M$. A constante $G$ é denominada de Constante da Gravitação Universal.

### 2. Aceleração Gravitacional

Na física, o campo de uma força é frequentemente mais importante que a magnitude absoluta da força. O campo é definido como a força exercida em uma unidade de material. Por exemplo, o campo elétrico de um corpo carregado em uma determinada posição é a força que ele exerce em uma unidade de carga elétrica naquele local. O **campo gravitacional** na vizinhança de uma massa de atração é a força que exerce sobre uma massa unitária. A Equação (1) mostra que isso é equivalente ao vetor de aceleração.

Em aplicações geofísicas, estamos preocupados com acelerações, e não com forças. Comparando Equação {1) e Equação (2) obtemos a aceleração gravitacional $a_G$ da massa $m$ devido à atração da massa $M$

\begin{equation}
    a_G = -G\frac{M}{r^2} \hat{\mathbf{r}}.  \label{acel_G} \tag{3}
\end{equation}

A unidade de aceleração do SI é o $(\mathrm{m}\mathrm{s}^{-2})$; esta unidade é impraticável para uso em geofísica. No agora superado sistema c.g.s.  a unidade de aceleração é de $(\mathrm{cm}\,\mathrm{s}^{-2})$, que é chamado de gal em reconhecimento das contribuições de Galileo. As pequenas mudanças na aceleração da gravidade causadas pelas estruturas geológicas são medidas em milésimos desta unidade, ou seja, em miligramas (mgal). Até recentemente, anomalias de gravidade devido a estruturas geológicas foram pesquisadas com instrumentos de campo com precisão de cerca de um décimo de miligal, o que foi chamado de **unidade de gravidade**. Instrumentos modernos são capazes de medir diferenças de gravidade para um milionésimo de gal, ou microgal $(\mu gal)$, que está se tornando a unidade prática de investigações de gravidade. O valor da gravidade na superfície da Terra é de cerca de $9.8\, \mathrm{m}\mathrm{s}^{-2}$, e assim a sensibilidade das medições modernas da gravidade é de cerca de 1 parte em $10^9$.

### 3. A Constante Graviatcional

A constante graviatcional é dado por $G=6,674210 \times 10^{-11} \mathrm{m}^3 \mathrm{kg}^{-1}\mathrm{s}^{-2}$.
uma forma de acessar essa constante é importar um pacote da biblioteca pyhton, o chamado é feito da seguinte forma a seguir:

In [1]:
from scipy.constants import G, g

In [2]:
print(G,g)

6.6743e-11 9.80665


#### Tabela 1: Valores característico da massa, forma e órbita dos planetas e da Lua

<table align="center">
<thead>
        <tr><th>Planeta</th><th>Massa M<br>$10^{24}~\mathrm{kg}$</th><th>Massa<br>relativa<br>a Terra<th>Densidade<br>média<br>[$\mathrm{km~m ^{-3}}$]</th><th>Raio<br>equatorial<br>$[\mathrm{km}]$</th>
    <th>Período<br>de rotação<br> sideral<br> $\mathrm{[dias]}$</th>
    <th>Achatamento inverso<br>$f^{-1}= a/(a-c)$</th>
    <th>Obliguidade do eixo<br> para orbita $(^\circ)$</th>
</thead>
<tbody>
<tr><td>Mercúrio</td><td>0.33011</td><td>0.0553</td><td>5427</td><td>2439.7</td><td>58.81</td><td> - </td><td>0.034</td></tr>
<tr><td>Vênus</td><td>4.8675</td><td>0.815</td><td>5243</td><td>6051.8</td><td>243.7</td><td> - </td>
<td>177.4</td></tr>
<tr><td>Terra</td><td>5.9723</td><td>1.000</td><td>5514</td><td>6378.1</td><td>0.9973</td><td>298.253</td>
<td>23.44</td></tr>
<tr><td>Lua</td><td>0.07346</td><td>0.0123</td><td>3344</td><td>1738.1</td><td>27.32</td><td>827.67</td>
<td>6.68</td></tr>
<tr><td>Marte</td><td>0.64171</td><td>0.1074</td><td>3933</td><td>3396.2</td><td>1.0275</td><td>169.81</td>
<td>25.19</td></tr>    
<tr><td>Júpiter</td><td>1899.19</td><td>317.83</td><td>1326</td><td>71492</td><td>0.414</td><td>15.41</td>
<td>3.13</td></tr>    
<tr><td>Saturno</td><td>568.34</td><td>95.16</td><td>687</td><td>60268</td><td>0.444</td><td>10.21</td>
<td>26.73</td></tr>
<tr><td>Urano</td><td>86.813</td><td>14.54</td><td>1271</td><td>25559</td><td>0.718</td><td>43.62.54</td>
<td>97.77</td></tr>
<tr><td>Netuno</td><td>102.413</td><td>17.15</td><td>1638</td><td>24764</td><td>0.671</td><td>58.54</td>
<td>28.32</td></tr>
<tr><td>Plutão</td><td>0.01303</td><td>0.00218</td><td>1854</td><td>1187</td><td>6.387</td><td> - </td>
<td>122.53</td></tr>    
</tbody>
</table>

### Exercício
1. Usando a tabela acima, calcule a aceleração gravitacional sobre as superfícies da Lua, Marte, Jupiter  e Vênus  como um percetual daquela da superfície da Terra.

**Solução:**

O módulo da aceleração gravitacional é dados por
\begin{equation}
    a_G = -G\frac{M}{r^2}.
\end{equation}
Para a Terra (Erath-e) podemos definir
\begin{equation}
    g_e = -G\frac{M_e}{r^2}. \tag{1.1}
\end{equation}

e para um planeta qualquer

\begin{equation}
    g_p = -G\frac{M_p}{r^2}. \tag{1.2}
\end{equation}

Dividindo  (1.2) por (1.1), teremos:

\begin{equation}
    g_p = g_e \frac{M_p}{M_e}. \tag{1.3}
\end{equation}

Anteriormente, vimos que a aceleração da gravidade na Terra é igual a $g_e=9.80665$, 
e que $\frac{Mp}{M_e}$ representa a massa relativa do planeta $p$ em relação a Terra $e$, ou seja
a coluna 2 da Tabela 1. Desse modo, complete os valores como no exemplo de Saturno a seguir

a) Saturno

In [3]:
a=g*95.16
print('aceleração gravitacional em Saturno é: ',a)

aceleração gravitacional em Saturno é:  933.2008139999999


Digite seu nome: Fulano de Tal, matricula:000000

#### <center>We love open source!</center>

<center><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" width=60 src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a> 

This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</center>